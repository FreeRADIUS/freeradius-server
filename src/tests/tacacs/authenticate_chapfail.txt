#
#	ARGV: -t chap -u scald authenticate -p pegapilha -i a -c qwerQWERtyuiTYUI
#
