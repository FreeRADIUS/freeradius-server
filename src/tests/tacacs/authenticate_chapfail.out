status: FAIL
data: b'Authentication-Data'
server_msg: b'CHAP authentication failed for scald'
